app/assets/javascripts/app.js

```diff
@@ -15,6 +15,11 @@ document.addEventListener("DOMContentLoaded", function(event) {
       $.get('/api/v1/leads.json').success(function(response) {
         console.log(this);
         this.leads = response;
+        this.leads = _.map(this.leads, function(lead){
+          lead.events = _.orderBy(lead.events, 'created_at', 'desc');
+          return lead;
+        })
+        this.leads = _.orderBy(this.leads, 'events[0].created_at', 'desc');
```

Comment:
Also, the other team came up with a totally different way of solving it. Look here: https://github.com/Actualize-Apprenticeship-Team-Gator/crm_api/pull/15/files

They used a rails method `touch` so anytime an event is created it will update the lead's `updated_at` timestamp. Then the index can be sorted without diving into nested arrays.